spec/features/merchants/items/index_spec.rb

```diff
@@ -41,9 +41,8 @@
 
     describe 'I see a button next to each item to enable or disable' do
       it 'where the button is enabled for disabled items' do
-        @item2.update(status: 1)
-        @item4.update(status: 1)
-        @item5.update(status: 1)
+        @item1.update(status: 1)
```

Comment:
Well our idea of a new item that is added should default to enabled was wrong, at least it was a quick fix.